tests/beta/queuing/test_beta_measure.py

```diff
@@ -30,10 +32,13 @@
     probs,
     Expectation,
     Sample,
+    State,
     Variance,
     Probability,
     MeasurementProcess
 )
+from pennylane.beta.tapes.qnode import QuantumFunctionError as QuantumFunctionErrorBeta  # TODO
```

Comment:
What's the intent here?

Reply:
This is now removed - we needed multiple versions of `QuantumFunctionError`: one comes from the core and one comes from the new beta content (although they are practically the same) - however the need for this was removed as we've taken out the torch version check.